src/app/pages/home-page/home-page.component.scss

```diff
@@ -1,10 +1,16 @@
+$background-content:rgba(37, 51, 73, 0.85);
```

Comment:
pls move colors to folder /src/app/shared/constats/colors.scss

Reply:
done